src/Media_Command.php

```diff
@@ -183,6 +183,10 @@ function regenerate( $args, $assoc_args = array() ) {
 	 * [--porcelain]
 	 * : Output just the new attachment ID.
 	 *
+	 * [--filetime]
```

Comment:
Can we make this flag more specific? Possibly `--use-filetime-for-date`?

See https://make.wordpress.org/cli/handbook/philosophy/#readability-trumps-number-of-keystrokes